app/controllers/api/internal/reviews_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::Internal::ReviewsController < Api::Internal::BaseController
+  before_action :authenticate_user!
+
+  def index
+    respond_with Review.all
+  end
+
+  def show
+    respond_with review
+  end
+
+  def create
+    respond_with Review.create!(review_params.merge(user: current_user))
+  end
+
+  private
+
+  def review
+    @review ||= Review.find_by!(id: params[:id])
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 19